### PR TITLE
Upgrade the app to Ember 3

### PR DESCRIPTION
**Pull request by blimmer**
I was getting warnings on my app that looked like this:

```
DEPRECATION: An addon is trying to access project.nodeModulesPath.
This is not a reliable way to discover npm modules. Instead, consider doing: 
require("resolve").sync(something, { basedir: project.root }). Accessed from:   new 
NPMDependencyVersionChecker (/Users/blimmer/code/ember-raffle/node_modules/ember-confetti
/node_modules/ember-cli-version-checker/src/npm-dependency-version-checker.js:11:33)
```

So I updated this app, which should prevent those warnings.

### CODE REVIEW

File: .bowerrc [hunk 1]
```diff
@@ -1,4 +0,0 @@
-{
```
Comment (blimmer) [inline .bowerrc:1]: Nothing bower-related was used

File: .eslintrc.js [hunk 2]
```diff
@@ -1,13 +1,49 @@
 module.exports = {
```
Comment (blimmer) [inline .eslintrc.js:1]: Everything below was from `ember init` unless otherwise called out.

File: addon/components/confetti-rain.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
-import Ember from 'ember';
-
-const { computed } = Ember;
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { htmlSafe } from '@ember/string';
+import { schedule, bind } from '@ember/runloop';
```
Comment (blimmer) [inline addon/components/confetti-rain.js:4]: Converted to module import

File: tests/integration/components/confetti-rain-test.js [hunk 4]
```diff
@@ -1,25 +1,27 @@
...
+    assert.equal(find('*').textContent.trim(), '');
 
-  // Template block usage:
-  this.render(hbs`
-    {{#confetti-rain}}
-      template block text
-    {{/confetti-rain}}
-  `);
+    // Template block usage:
+    await render(hbs`
```
Comment (blimmer) [inline tests/integration/components/confetti-rain-test.js:19]: Converted to the new async/await helpers

File: tests/integration/components/confetti-rain-test.js [hunk 5]
```diff
@@ -1,25 +1,27 @@
...
+      {{#confetti-rain}}
+        template block text
+      {{/confetti-rain}}
+    `);
 
-  assert.equal(this.$().text().trim(), 'template block text');
+    assert.equal(find('*').textContent.trim(), 'template block text');
```
Comment (blimmer) [inline tests/integration/components/confetti-rain-test.js:25]: Native DOM vs. jQuery

**Comment by san650** [comment]
@blimmer this is awesome!! Thanks for taking the time to fix the issue.

I'll release a new version later today.

**Comment by san650** [comment]
@blimmer published as v1.0.4. Thanks again for contributing!!

**Pull request merged**